src/activeobject.h

```diff
@@ -64,6 +64,7 @@ enum ActiveObjectCommand {
 	AO_CMD_UPDATE_ARMOR_GROUPS,
 	AO_CMD_SET_ANIMATION,
 	AO_CMD_SET_BONE_POSITION,
+	AO_CMD_UNSET_BONE_POSITION,
```

Comment:
Adding something in the middle of this enum will change the values, breaking backwards compatibility.